specification/subscription/resource-manager/Microsoft.Subscription/preview/2024-08-01-preview/subscriptions.json

```diff
@@ -753,6 +753,144 @@
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "parameters": [
+          {
+            "$ref": "#/parameters/transferIdParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "description": "Transfer creation request.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/TransferCreationRequest"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Transfer Details.",
+            "schema": {
+              "$ref": "#/definitions/TransferDetails"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v5/types.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      },
+      "get": {
+        "description": "Retrieves transfer details using a system-generated TransferId. ",
+        "operationId": "Transfers_GetById",
+        "x-ms-examples": {
+          "GetTransferById": {
+            "$ref": "./examples/getTransfer.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/transferIdParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Transfer details.",
+            "schema": {
+              "$ref": "#/definitions/TransferDetails"
+            }
+          },
+          "default": {
+            "description": "Describes the error if the operation is not successful.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v5/types.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      },
+      "delete": {
+        "description": "Cancel the transfer by using its system generated TransferId.",
+        "operationId": "Transfers_CancelById",
+        "x-ms-examples": {
+          "GetTransferById": {
+            "$ref": "./examples/cancelTransfer.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/transferIdParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Transfer details.",
+            "schema": {
+              "$ref": "#/definitions/TransferDetails"
+            }
+          },
+          "default": {
+            "description": "Describes the error if the operation is not successful.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v5/types.json#/definitions/ErrorResponse"
+            }
```

Comment:
>/providers/Microsoft.Subscription/subscriptions/{subscriptionId}/transfers/default [](http://example.com/codeflow?start=5&length=82)

why is this API needed if 
1. existing GET API
2. Proposed async POST actions' polling headers 
show the status anyway ?

Reply:
@abdulgagit Please address this.

Reply:
This is used to get the latest transfer, to fetch the system generated transfer id